README.md

```diff
@@ -1,2 +1,47 @@
-# zipkin-sparkstreaming-example
-Example of how to make a custom adjuster for zipkin-sparkstreaming
+# zipkin-sparkstreaming custom adjuster example
+
+This example shows how to add a custom adjuster for zipkin-sparkstreaming.
+We'll create a jar with a simple adjuster that trims the length of binary annotation values.
+This adjuster will be applied by placing it in the class path of the spark job.
+
+## Steps for creating an adjuster jar
```

Comment:
For the readme to help people get started, the primary goal is "how to use this", not which steps maven is doing.

For example, in Brave example, we don't tell people how the maven plugin works to make a war file.